WordPress/build.gradle

```diff
@@ -199,6 +196,12 @@ android {
 
     flavorDimensions "app", "buildType"
 
+    Map<String, Object> secretProperties = loadSecrets(
+            project.logger,
+            file("${rootDir}/defaults.properties"),
+            file("${System.getProperty("user.home")}/.configure/wordpress-android/secrets/secrets.properties")
```

Comment:
Praised (❤️): Nicely done, better be safer! 🥇

Reply:
Kudos to @mokagio for this suggestion! 🙌